src/core/lv_obj_tree.h

```diff
@@ -166,11 +174,20 @@ uint32_t lv_obj_get_child_cnt(const struct _lv_obj_t * obj);
 
 /**
  * Get the index of a child.
- * @param obj       pointer to an obejct
+ * @param obj       pointer to an object
+ * @return          the child index of the object.
+ *                  E.g. 0: the oldest (firstly created child)
+ */
+uint32_t lv_obj_get_index(const struct _lv_obj_t* obj);
+
+/**
+ * DEPRECATED FUNCTION, please use uint32_t lv_obj_get_index(obj);
+ * Get the index of a child.
+ * @param obj       pointer to an object
  * @return          the child index of the object.
  *                  E.g. 0: the oldest (firstly created child)
  */
-uint32_t lv_obj_get_child_id(const struct _lv_obj_t * obj);
+uint32_t lv_obj_get_child_id(const struct _lv_obj_t* obj);
```

Comment:
Making it `static inline` in this header instead of `lv_api_map.h` should work.

Reply:
that did work, thanks